client/src/js/saved-places.js

```diff
@@ -61,20 +61,23 @@ const makePinMarkers = (pinArray = [], L, markerOptions=FAV_MARKER_OPTIONS) => {
         draggable: false,
         autopan: true,
         riseOnHover: true,
-        title: pin.place_id
+        title: pin.place_id,
+        desc: pin.desc
       });
 
-    } else { 
+    } else {
       thisMarker =  L.marker([pin.lat, pin.lng], {
         draggable: false,
         autopan: true,
         riseOnHover: true,
-        title: pin.place_id
+        title: pin.place_id,
+        desc: pin.desc
       });
     }
-    
-    thisMarker.data = pin; //save the db data to the marker
 
+    const container = L.DomUtil.create('div');
```

Comment:
`container` is unused within the scope of `makePinMarkers()` - it looks like it should be deleted? It looks like it actually implemented at line 112.

Reply:
You would be correct. I think I originally had logic there and had to move it down. Great catch.